src/workspace.js

```diff
@@ -1006,6 +1170,64 @@ module.exports = class Workspace extends Model {
     if (this.activeItemSubscriptions != null) {
       this.activeItemSubscriptions.dispose()
     }
+    if (this.movedItemSubscription != null) {
+      this.movedItemSubscription.dispose()
+    }
+  }
+
+  /*
+  Section: Pane Locations
+  */
+
+  getCenter () {
+    return this.center
+  }
+
+  getLeftDock () {
+    return this.docks.left
+  }
+
+  getRightDock () {
+    return this.docks.right
+  }
+
+  getBottomDock () {
+    return this.docks.bottom
+  }
+
+  getPaneLocations () {
```

Comment:
The term `location` is used to refer both to strings like `left` and `center` and also to objects like `Docks` and the `WorkspaceCenter`. @nathansobo and I think that when referring generically to these objects, we should call them 'pane containers'. There's some ambiguity with this name, because `PaneContainer` is an internal class which all of these classes *wrap*, but we think this is ok, because these classes have a kind of *is a* relationship with the `PaneContainer` class, even though we use composition rather than inheritance. Also, the `PaneContainer` is a private class, whereas the name `location` shows up in our API in methods like `getDefaultLocation`.